docs/content/posts/open-source-introduction.md

```diff
@@ -0,0 +1,204 @@
+---
+date: 2019-12-26
+title: Open source
+tags: [open-source]
+---
+
+# Open source: CI/CD and test infrastructure for Android
+
+Avito.ru is the biggest Russian classified. 
+We have moved our Android infrastructure into open source: Gradle plugins, emulators, and test libraries. 
+[Our code](https://github.com/avito-tech/avito-android) will be useful in automating CI/CD and will also facilitate the coding and support of autotests. 
+ 
+In this review, we will explain why we decided to move into open source, present the central libraries of the project, and suggest whom to contact with any questions. 
+We will analyze in detail the individual libraries, Gradle plugins, and our development approaches in future posts.
+ 
+![ijhasr2dv95azaem9mgilu7e1kw](https://user-images.githubusercontent.com/1104540/80410956-0cf44200-88d4-11ea-879d-cd9b6acd29af.png)
+
+## Who we are and what we do
+
+We are developing solutions for Android as part of the Speed platform team. There are four of us:
+
+{{< columns >}}
+
+Sergey Boishtyan\
+Senior engineer\
+[sboishtyan](https://twitter.com/sboishtyan)
+
+![](https://user-images.githubusercontent.com/1104540/80411459-e8e53080-88d4-11ea-9751-1ed675da9f25.jpg)
+
+<--->
+
+Dima Voronin\
+Lead engineer\
+[DmitriVoronin](https://twitter.com/DmitriVoronin)
+
+![](https://user-images.githubusercontent.com/1104540/80411540-074b2c00-88d5-11ea-8087-ddbab5d85dd3.jpg)
+
+<--->
+
+Eugene Krivobokov\
+Senior engineer\
+[eugene_kr](https://twitter.com/eugene_kr)
+
+![](https://user-images.githubusercontent.com/1104540/80413105-9a856100-88d7-11ea-9e07-b5a5e48882cf.png)
+
+<--->
+
+Daniil Popov\
+Senior engineer\
+[Int02h](https://twitter.com/Int02h)
```

Comment:
I don't think Twitter links are case-sensitive but I'd change this to `int02h` with the first letter in lower-case.

Reply:
They are case-insensitive. Ok, I'll change